docs/frontend/sortable-list.md

```
@@ -0,0 +1,192 @@
+        </div>
+    )}
+    ...
+/>
+```
+
+## Important Dependencies
+For both `Dokan Free and Pro` versions, we must register the `dokan-react-components` dependency when using `global` components.
+
+## Quick Overview
+
+#### Step 1: Import the Required Components
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)